src/pk_compiler.c

```diff
@@ -1853,6 +1857,7 @@ static void emitAssignment(Compiler* compiler, TokenType assignment) {
     case TK_ANDEQ: emitOpcode(compiler, OP_BIT_AND); break;
     case TK_OREQ: emitOpcode(compiler, OP_BIT_OR); break;
     case TK_XOREQ: emitOpcode(compiler, OP_BIT_XOR); break;
+    case TK_MODEQ: emitOpcode(compiler, OP_MOD); break;
```

Comment:
Here is the order is wrong, but It's not that a big deal, I'll merge it and change myself.